Backend/router/v1/user.js

```diff
@@ -13,15 +13,34 @@ router.all('/', (req, res) => {
 });
 
 router.post('/signup', (req, res) => {
+
   let hashPassword = crypto.createHash("sha512").update(req.body.password).digest("hex");
```

Comment:
hashedPassword (hashPassword means you will hash the password, which is not this case)

Reply:
hashedPassword -> encryption , Is it ok?

Reply:
I don't think encryption is a good variable name. It doesn't tell us what the variable holds. Why did you decided to make this change? hashedPassword seemed good go me (it reflects what the variable value means, right?).

Reply:
Oh, I didn't understand your comment. Now I understand it. I will change to hashedPassword. Thank you! :)